src/app/code/community/Meanbee/ConfigPoweredCss/Model/Config.php

```diff
@@ -77,6 +78,66 @@ public function getCssFilename($store = null)
      */
     public function getFullCssFilePath($store = null)
     {
-        return Mage::getBaseDir('skin') . '/frontend/base/default/' . $this->getCssFilename($store);
+        return $this->getCssDirectoryPath() . $this->getCssFilename($store);
+    }
+
+    /**
+     * Get directory for CSS files
+     *
+     * @param $store
+     * @return string
+     */
+    public function getCssDirectoryPath($store = null)
+    {
+
+        return Mage::getBaseDir('media') . $this->getCssPath($store);
+    }
+
+    /**
+     * Retrieve the CSS path relative to the media directory
+     * @param null $store
+     * @return string
+     */
+    public function getCssPath($store = null)
+    {
+        return sprintf(self::CSS_PATH, $this->_getDesignPackage($store), $this->_getDesignTheme($store));
+    }
+
+    /**
+     * Get full URL for CSS file
+     * @param null $store
+     * @return string
+     */
+    public function getCssFileUrl($store = null)
+    {
+        return Mage::getUrl(sprintf('media/%s', $this->getCssPath($store))) . $this->getCssFilename($store);
```

Comment:
We should fetch the media url properly: `Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_MEDIA);`

If you're using a CDN, currently the URL will only be the base url of the store.

Also, it's possible SSID parameters to be included in this URL.